utils/chunked_vector.hh

```diff
@@ -471,7 +471,7 @@ template <typename T, size_t max_contiguous_allocation>
 void chunked_vector<T, max_contiguous_allocation>::set_chunk_after_emplace_back(std::pair<chunk_ptr, size_t> x) noexcept {
     auto new_chunk_ptr = std::move(x.first);
     auto new_chunk_capacity = x.second;
-    // If the new chunk os replacing the first chunk, migrate the existing elements onto it.
+    // If the new chunk is replacing the first chunk, migrate the existing elements onto it.
```

Comment:
@tchaikov while at it, can you also fix the `capcity` mispelling below?

Reply:
yeah, thanks! fixed and repushed.